Library/Formula/plan9port.rb

```diff
@@ -17,15 +17,10 @@ class Plan9port < Formula
   def install
     ENV["PLAN9_TARGET"] = libexec
 
-    if build.with? "x11"
-      # Make OS X system fonts available to Plan 9
-      File.open("LOCAL.config", "a") do |f|
-        f.puts "FONTSRV=fontsrv"
-      end
-    end
+    # Make OS X system fonts available to Plan 9
+    (buildpath/"LOCAL.config").write "FONTSRV=fontsrv" if build.with? "x11"
```

Comment:
Thanks for the clarification! I hadn't tested this myself and found this to be rather odd in the context of this PR. (Good thing there is this check for existence/non-existence of the target file that helps to avoid subtle mistakes like this one could have been!)